brand-guidelines/color-principles.md

```diff
@@ -24,945 +24,109 @@ A minimal palette, clear hierarchy, good information design, and ample whitespac
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>173, 220, 145</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>358</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>32, 0, 52, 0</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-    <figure class="swatch swatch__primary">
-        <div class="swatch_field swatch_field__green-tint"></div> <!-- Update the color in the class name -->
-        <figcaption class="swatch_label">
-            <h4 class="swatch_head">Green Tint</h4> <!-- Update the full color name -->
-            <table class="swatch_table">
-                <tbody>
-                    <tr>
-                        <th>HEX</th>
-                        <td>#DBEDD4</td> <!-- Update the hex code -->
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>219, 237, 212</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>361 @ 20%</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>14, 0, 20, 0</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-    <figure class="swatch swatch__primary">
-        <div class="swatch_field swatch_field__black"></div> <!-- Update the color in the class name -->
-        <figcaption class="swatch_label">
-            <h4 class="swatch_head">CFPB Black</h4> <!-- Update the full color name -->
-            <table class="swatch_table">
-                <tbody>
-                    <tr>
-                        <th>HEX</th>
-                        <td>#101820</td> <!-- Update the hex code -->
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>16, 24, 32</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>Black 6</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>100, 79, 44, 93</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-</div> <!-- /.swatches -->
+    {% for color in site.data.cfpb-brand-colors %}
+        {% if color.primary == "true" %}
```

Comment:
My only worry is if someone added an entry there by accident, but I can change it.